src/reducers/cartReducer.js

```diff
@@ -21,12 +21,14 @@ const cartReducer = (state = initialState, action) => {
         case ADD_TO_CART:
             return {
                 ...state,
-                addedItems: [...state.addedItems, action.addedItem],
+                addedItems: [...state.addedItems, action.payload.addedItem],
+                total: action.payload.newTotal
```

Comment:
const { type, payload = {} } = action;
// ADD TO CART
const { newTotal = 0 } = payload;